src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  @include circle($stopwatch-size);
+  margin: 0 auto;
+  border: 5px solid $stopwatch-color;
+  cursor: pointer;
+
+  &:hover .animated {
```

Comment:
strange logic)

Reply:
Why?

Reply:
As it is stopwatch, the logic is that we should stop it at any moment. 
So, I decided to realise that in this way (when we hover on this stopwatch, the animation stoped). That's why I implemented cursor:pointer, because user shoud see, that this element is interactive

Reply:
What's wrong with this logic? 
Explain your comment, plz

Reply:
better to have some button to toggle start, it's not obvious)